### PR TITLE
seperated readonly prop to TextInput, IconButton #461

### PR DESCRIPTION
**Pull request by mahmoud-zakria-sociomantic**
#461

### CODE REVIEW

File: src/TextInputWithIcon/index.jsx [hunk 1]
```diff
@@ -140,6 +140,14 @@ class TextInputWithIcon extends Component
...
          */
         isReadOnly            : PropTypes.bool,
+        /**
+         *  Display as read-only for TextInput
+         */
+        isReadOnlyInput       : PropTypes.bool,
+        /**
+         *  Display as read-only for IconButton
+         */
+        isReadOnlyBtn         : PropTypes.bool,
```
Comment (conor-cafferkey-sociomantic) [inline src/TextInputWithIcon/index.jsx]: Generally we don’t abbreviate `Button` 👉 `Btn`.  So let’s change this to `isReadOnlyButton` to be consistent with the naming used in other components.

Comment (mahmoud-zakria-sociomantic) [reply src/TextInputWithIcon/index.jsx]: Sure 👍

Comment (mahmoud-zakria-sociomantic) [reply src/TextInputWithIcon/index.jsx]: Done, Checking...

Comment (mahmoud-zakria-sociomantic) [reply src/TextInputWithIcon/index.jsx]: Passed.

**Pull request merged**